src/routes/transactions/mappers/multisig-transactions/transaction-info/erc721-transfer.mapper.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@nestjs/common';
+import { MultisigTransaction } from '../../../../../domain/safe/entities/multisig-transaction.entity';
+import { Token } from '../../../../../domain/tokens/entities/token.entity';
+import { AddressInfoHelper } from '../../../../common/address-info/address-info.helper';
+import { TransferTransactionInfo } from '../../../entities/transfer-transaction-info.entity';
+import { Erc721Transfer } from '../../../entities/transfers/erc721-transfer.entity';
+import { DataDecodedParamHelper } from './data-decoded-param.helper';
+import { TransferDirectionHelper } from './transfer-direction.helper';
+
+@Injectable()
+export class Erc721TransferMapper {
+  private static readonly NULL_ADDRESS =
+    '0x0000000000000000000000000000000000000000';
```

Comment:
We should extract this. It's already being used in a couple places.

Reply:
Included in [d985d6c](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/d985d6cb5043fb4c54881c2cdab35c7007f20306) 👍🏻